unblob/extractor.py

```diff
@@ -139,4 +124,4 @@ def carve_valid_chunk(
     carve_path = extract_dir / filename
     logger.info("Extracting valid chunk", path=carve_path, chunk=chunk)
     carve_chunk_to_file(carve_path, file, chunk)
-    return carve_path
+    return carve_path.resolve()
```

Comment:
why resolve it here? The carved file is created in the previous line, why do any resolve after?

also `carve_chunk_to_file` requires absolute path, so most likely it is already resolved

Reply:
yes, indeed, on line 139 `extract_dir` is used, which should be an absolute path, so `resolve` is not needed.

actually it should not really matter if we are using absolute or relative paths, as long as the input ones are pointing to the input file and the extraction root can be written to/created.

Reply:
removed